### DISCUSSION TITLE
[求助]小邦布到底说了些什么？哪位绳匠来解密一下！

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#56] [其他]
![Cover](https://i0.hdslb.com/bfs/archive/67048ddc4c18c160c8d6c7db1d0fe35c6075276d.jpg)
各位绳匠们，小邦布发表了重要讲话！可惜没有翻译看不懂！邦布到底说了什么？解密有奖！解密成功者赏114514丁尼！

原文：
！嗯？！呢？嗯嗯嗯？？嗯嗯嗯嗯呢！嗯嗯？嗯嗯嗯？？嗯呢？呢？嗯？嗯嗯呢？嗯？？！嗯嗯呢？呢！！呢嗯？嗯嗯嗯？！嗯嗯嗯呢！呢嗯嗯！嗯！！呢？嗯呢？嗯？呢嗯？嗯呢？呢嗯！嗯？嗯呢？嗯呢！嗯嗯？嗯嗯？嗯呢？！呢嗯呢呢！？？嗯嗯！嗯！嗯？呢嗯嗯！嗯！！？呢嗯嗯！嗯？！？呢呢嗯嗯嗯嗯呢呢呢呢？呢呢呢嗯嗯嗯呢呢！？嗯嗯？？呢嗯嗯！嗯？？？？呢嗯嗯？？呢！！嗯！！！？嗯嗯！嗯呢嗯嗯呢嗯嗯！嗯呢？呢？嗯嗯！嗯？嗯？呢嗯嗯！嗯嗯呢呢呢嗯嗯！嗯？嗯呢？嗯嗯！嗯嗯？呢呢嗯嗯嗯嗯？呢呢？呢嗯嗯呢呢呢嗯！嗯！？呢？嗯嗯嗯嗯！呢嗯呢呢嗯！！嗯？嗯嗯嗯嗯！嗯？？！！嗯嗯嗯！嗯呢呢！嗯嗯嗯！嗯？嗯嗯？呢嗯？呢呢呢！嗯呢！！？嗯？！呢呢？呢呢！！？？呢？？呢呢嗯！？嗯嗯！嗯嗯！呢！？？？！！？呢嗯！！！！！呢！！！？！！？！！！呢呢？呢！？呢呢呢呢！？！嗯呢嗯！！？嗯嗯呢！！嗯嗯嗯嗯

*嗯呢？！*

> 绳匠批注：这不是乱打的！是真的隐藏了重要信息[doge]！

**Comment by BiaoBiaoB** [comment]
嗯呐，嗯呐嗯呐！（看看，看看，这就是家长不给安装好的语言插件的下场！）
嗯呐嗯呐嗯呐。（接下来我来翻译一下。）
！你说什么？！啊？你说这个？这东西竟然是真的！我要去，我要去！
额，我家邦布忽然就跑了出去，一边跑，一边说着什么黄金邦布的？桌面上就剩下了这么个网页，所以谁能告诉我发生了什么？

**Comment by xiangxliuying** [comment]
它想玩原神了
